src/pip/_internal/commands/install.py

```diff
@@ -616,11 +616,13 @@ def decide_user_install(
     If use_user_site is None, the default behaviour depends on the environment,
     which is provided by the other arguments.
     """
-    if use_user_site is False:
+    # In some cases (config from tox), use_user_site can be set to an integer
```

Comment:
Also it can be reproduced outside of tox. Add `user = true` to `pip.conf` and you'll get the same error on `pip install six`. That happens because `ConfigOptionParser` uses `distutils.util.strtobool()` when parses values from config file: https://github.com/pypa/pip/blob/c729a84b4884511912f459f3e340367c2872f495/src/pip/_internal/cli/parser.py#L201

distutils.util:
```python
def strtobool (val):
    """Convert a string representation of truth to true (1) or false (0).

    True values are 'y', 'yes', 't', 'true', 'on', and '1'; false values
    are 'n', 'no', 'f', 'false', 'off', and '0'.  Raises ValueError if
    'val' is anything else.
    """
    val = val.lower()
    if val in ('y', 'yes', 't', 'true', 'on', '1'):
        return 1
    elif val in ('n', 'no', 'f', 'false', 'off', '0'):
        return 0
    else:
        raise ValueError("invalid truth value %r" % (val,))
```

It could be worth to comment the case with config file.

Reply:
Would be nice to have a test fot that case, btw.

Reply:
Like this one, but `install`:
https://github.com/pypa/pip/blob/c729a84b4884511912f459f3e340367c2872f495/tests/functional/test_wheel.py#L221-L229

Reply:
Thanks, copied part of that and added a test.